crates/bevy_tasks/src/task_pool.rs

```diff
@@ -31,13 +31,13 @@ impl TaskPoolBuilder {
 
     /// Override the number of threads created for the pool. If unset, we default to the number
     /// of logical cores of the system
-    pub fn num_threads(mut self, num_threads: usize) -> Self {
+    pub const fn num_threads(mut self, num_threads: usize) -> Self {
         self.num_threads = Some(num_threads);
         self
     }
 
     /// Override the stack size of the threads created for the pool
-    pub fn stack_size(mut self, stack_size: usize) -> Self {
+    pub const fn stack_size(mut self, stack_size: usize) -> Self {
```

Comment:
`TaskPoolBuilder` does not have a const-constructor, but the `new` method could be made trivially const.